### ISSUE TITLE
`editor.accessibilitySupport` should be migrated to `accessibility.support` or similar

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#21751]
`editor.accesssibilitySupport` is used all over the place at this point. We should migrate to a new setting with a name that correctly reflects the broad scope to which it applies.
